src/iterative_ensemble_smoother/experimental.py

```diff
@@ -36,6 +36,7 @@ def ensemble_smoother_update_step_row_scaling(
             D,
             inversion,
             truncation,
+            True,
```

Comment:
should we do projection when row_scaling? I thought that was doing update one parameter at a time?

Reply:
Good point, thanks.